### ISSUE TITLE
Inconsistency with ACT aria-hidden focusable rule

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#3406]
Axe-core is misreporting the following examples 

**[Passed Example 4](https://act-rules.github.io/rules/6cfa84#passed-example-4)**

```html
<div aria-hidden="true">
	<a href="/" style="position:absolute; top:-999em" onfocus="document.querySelector('input').focus()">First link</a>
</div>
<input />
```

**[Failed Example 5](https://act-rules.github.io/rules/6cfa84#failed-example-5)**

```html
<div aria-hidden="true">
	<button tabindex="-1">Some button</button>
</div>
```

I disagree that failed example 5 should fail. I opened an issue for that here https://github.com/act-rules/act-rules.github.io/issues/1807 But I don't think we should be failing passed example 4. We can't accurate detect what event listeners exist (https://github.com/whatwg/dom/issues/412) but we can at least look for the onfocus property that is used in that example.
